README.md

```diff
@@ -39,44 +24,95 @@ nodes. Features:
-| `HEAD`   | ✕                  | ✓                | ✓               | ✓               | ✓               | ✓               | ✓               | ✓               | ✓               | ✓               | ✓               | ✓               | ✓               |
-
-## Install
-
-Please refer to our [Quickstart](https://github.com/openebs/dynamic-localpv-provisioner/blob/develop/docs/quickstart.md) and the [OpenEBS Documentation](http://openebs.io/docs/).
-
-## Contributing
+### Why OpenEBS Dynamic LocalPV Provisioner?
 
-Head over to the [CONTRIBUTING.md](./CONTRIBUTING.md) page.
-
-## Roadmap
-
-Find the Dynamic Local PV roadmap items at the [OpenEBS Roadmap page](https://github.com/openebs/openebs/blob/HEAD/ROADMAP.md#dynamic-local-pvs).
-
-## OpenEBS Adopters
-
-Check out the list of organizations and users who have chosen OpenEBS to run their stateful workloads, over at the [OpenEBS Adopters page](https://github.com/openebs/openebs/blob/HEAD/ADOPTERS.md).
-
-## Community, discussion, and support
+Local Persistent Volumes are great for distributed cloud native data services that can handle resiliency and availability and expect low-latency access to the storage. Local Persistent Volumes can be provisioned using the hostpath, NVMe or PCIe based SSDs, Hard Disks or on top of other filesystems like ZFS, LVM.
 
-Learn how to engage with the OpenEBS community on the [community page](https://github.com/openebs/openebs/tree/HEAD/community).
+Some of the targetted applications are:
 
-You can reach the maintainers of this project at:
+- Distributed SQL Databases like PostgreSQL
+- Distributed No-SQL Databases like MongoDB, Cassandra
+- Distributed Object Storages like MinIO (distributed mode)
+- Distributed Streaming services like Apache Kakfa,
+- Distributed Logging and search services like ElasticSearch, Solr
+- AI/ML workloads
 
-- [Kubernetes Slack](http://slack.k8s.io/) channels:
-  - [#openebs](https://kubernetes.slack.com/messages/openebs/)
-  - [#openebs-dev](https://kubernetes.slack.com/messages/openebs-dev/)
-- [Mailing List](https://lists.cncf.io/g/cncf-openebs-users)
+# Architecture
+
+```mermaid
+
+graph TD
+    %% Define Styles with Black Text
+    classDef kubelet fill:#ffcc00,stroke:#d4a017,stroke-width:2px,color:#000
+    classDef provisioner fill:#66ccff,stroke:#3388cc,stroke-width:2px,color:#000
+    classDef helper fill:#99ff99,stroke:#44aa44,stroke-width:2px,color:#000
+    classDef app fill:#ff9999,stroke:#cc6666,stroke-width:2px,color:#000
+    classDef storage fill:#ffdd99,stroke:#d4a017,stroke-width:2px,color:#000
+    classDef persistent fill:#d9b3ff,stroke:#9955cc,stroke-width:2px,color:#000
+
+    subgraph "Kubernetes Cluster"
+        subgraph "Kubelet"
+            KUB["Kubelet"]:::kubelet
```

Comment:
I haven't seen this syntax before, wouldn't this work?

```suggestion
    style kubelet fill:#ffcc00,stroke:#d4a017,stroke-width:2px,color:#000
    classDef provisioner fill:#66ccff,stroke:#3388cc,stroke-width:2px,color:#000
    classDef helper fill:#99ff99,stroke:#44aa44,stroke-width:2px,color:#000
    classDef app fill:#ff9999,stroke:#cc6666,stroke-width:2px,color:#000
    classDef storage fill:#ffdd99,stroke:#d4a017,stroke-width:2px,color:#000
    classDef persistent fill:#d9b3ff,stroke:#9955cc,stroke-width:2px,color:#000

    subgraph "Kubernetes Cluster"
        subgraph "Kubelet"
            kubelet
```

Reply:
Yeah yeah that also works, I have changed the whole thing to that way now.